doc/connectivity/bluetooth/api/audio/shell/bap.rst

```diff
@@ -32,6 +32,7 @@ Commands
       config                 : <direction: sink, source> <index> [loc <loc_bits>]
                               [preset <preset_name>]
       stream_qos             : interval [framing] [latency] [pd] [sdu] [phy] [rtn]
+      stream_connect         : Connect the CIS of the stream
```

Comment:
Maybe it would be good to show this call in a small example (with complete flow)